src/3rdParty/salomesmesh/src/SMDS/SMDS_UnstructuredGrid.cpp

```diff
@@ -1026,7 +1026,12 @@ void SMDS_UnstructuredGrid::BuildLinks()
   GetLinks()->Allocate(this->GetNumberOfPoints());
   GetLinks()->Register(this);
 //FIXME: vtk9
+  #if VTK_VERSION_NUMBER < VTK_VERSION_CHECK(9,3,0)
```

Comment:
Where are VTK_VERSION_NUMBER and VTK_VERSION_CHECK are defined?

Reply:
https://kitware.github.io/paraview-docs/latest/cxx/vtkVersionMacros_8h.html

I'm not sure whre it gets included but compilation suceeded for me on arch and on ubuntu

Reply:
Hhhmmm I'm not finding these on 7.1.1 source code though

Reply:
yeah seems it was added in vtk 8 https://gitlab.kitware.com/vtk/vtk/-/commit/2b94edcd2b74fb6f523c96a01c6b930313053215